### PR TITLE
Migrate org.gnome.gitlab.somas.Apostrophe.Plugin.TexLive

### PR DESCRIPTION
**Pull request by somas95**

### CODE REVIEW

File: org.gnome.gitlab.somas.Apostrophe.Plugin.TexLive.appdata.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+
```
Comment (bilelmoussaoui) [inline org.gnome.gitlab.somas.Apostrophe.Plugin.TexLive.appdata.xml:1]:
```suggestion
```

**Comment by bilelmoussaoui** [comment]
bot, build org.gnome.gitlab.somas.Apostrophe.Plugin.TexLive

**Comment by somas95** [comment]
bot, build org.gnome.gitlab.somas.Apostrophe.Plugin.TexLive

**Comment by somas95** [comment]
bot, build org.gnome.gitlab.somas.Apostrophe.Plugin.TexLive

**Comment by somas95** [comment]
bot, build org.gnome.gitlab.somas.Apostrophe.Plugin.TexLive

**Comment by somas95** [comment]
bot, build org.gnome.gitlab.somas.Apostrophe.Plugin.TexLive

File: org.gnome.gitlab.somas.Apostrophe.Plugin.TexLive.json [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    "branch": "stable",
+    "runtime-version": "stable",
+    "sdk": "org.gnome.Sdk//3.36",
+    "build-extension": true,
+    "separate-locales": false,
+    "appstream-compose": false,
+    "finish-args": [],
+    "build-options" : {
+        "prepend-path": "/app/extensions/TexLive",
+        "cflags": "-O2 -g",
```
Comment (bilelmoussaoui) [inline org.gnome.gitlab.somas.Apostrophe.Plugin.TexLive.json:13]:
```suggestion
```
already set by default

File: org.gnome.gitlab.somas.Apostrophe.Plugin.TexLive.json [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+        "cxxflags": "-O2 -g",
```
Comment (bilelmoussaoui) [inline org.gnome.gitlab.somas.Apostrophe.Plugin.TexLive.json:14]:
```suggestion
```
same thing

**Pull request closed without merging**